theme.config.js

```diff
@@ -1,67 +1,98 @@
-export default {
-  github: 'https://github.com/shuding/nextra',
-  docsRepositoryBase: 'https://github.com/shuding/nextra/blob/master',
+import { useRouter } from "next/router";
+
+const theme = {
+  github: "https://github.com/shuding/nextra",
+  projectLink: "https://github.com/shuding/nextra",
+  docsRepositoryBase:
+    'https://github.com/shuding/nextra/tree/core/examples/docs/pages',
   titleSuffix: ' – Nextra',
-  logo: (
-    <>
-      <span className="mr-2 font-extrabold hidden md:inline">Nextra</span>
-      <span className="text-gray-600 font-normal hidden md:inline">
-        The Next.js Static Site Generator
-      </span>
-    </>
-  ),
-  head: (
-    <>
-      <meta name="msapplication-TileColor" content="#ffffff" />
-      <meta name="theme-color" content="#ffffff" />
-      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-      <meta httpEquiv="Content-Language" content="en" />
-      <meta name="description" content="Nextra: the Next.js site builder" />
-      <meta name="og:description" content="Nextra: the Next.js site builder" />
-      <meta name="twitter:card" content="summary_large_image" />
-      <meta name="twitter:image" content="https://nextra.vercel.app/og.png" />
-      <meta name="twitter:site:domain" content="nextra.vercel.app" />
-      <meta name="twitter:url" content="https://nextra.vercel.app" />
-      <meta name="og:title" content="Nextra: Next.js static site generator" />
-      <meta name="og:image" content="https://nextra.vercel.app/og.png" />
-      <meta name="apple-mobile-web-app-title" content="Nextra" />
-      <link
-        rel="apple-touch-icon"
-        sizes="180x180"
-        href="/apple-icon-180x180.png"
-      />
-      <link
-        rel="icon"
-        type="image/png"
-        sizes="192x192"
-        href="/android-icon-192x192.png"
-      />
-      <link
-        rel="icon"
-        type="image/png"
-        sizes="32x32"
-        href="/favicon-32x32.png"
-      />
-      <link
-        rel="icon"
-        type="image/png"
-        sizes="96x96"
-        href="/favicon-96x96.png"
-      />
-      <link
-        rel="icon"
-        type="image/png"
-        sizes="16x16"
-        href="/favicon-16x16.png"
-      />
-      <meta name="msapplication-TileImage" content="/ms-icon-144x144.png" />
-    </>
-  ),
   search: true,
-  prevLinks: true,
-  nextLinks: true,
+  unstable_flexsearch: true,
+  unstable_faviconGlyph: '👋',
+  unstable_staticImage: true,
+  floatTOC: true,
+  font: false,
+  projectChatLink: "https://discord.gg/hEM84NMkRv",
+  feedbackLink: "Question? Give us feedback →",
+
+  logo: function LogoActual() {
+    return (
+      <>
+        <span className="font-extrabold">Nextra</span>
+        <span className="mr-2 ml-2 text-gray-500 font-normal hidden md:inline">
+          The Next Site Builder
+        </span>
+      </>
+    );
+  },
```

Comment:
Why did you refactor it to react component? what is the purpose?
```suggestion
  logo: (
      <>
        <span className="font-extrabold">Nextra</span>
        <span className="mr-2 ml-2 text-gray-500 font-normal hidden md:inline">
          The Next Site Builder
        </span>
      </>
    );
```